src/fsharp/ExtensionTyping.fs

```diff
@@ -520,7 +521,9 @@ module internal ExtensionTyping =
                     // To allow a type provider to depend only on FSharp.Core 4.3.0.0, it can alternatively implement an appropriate method called GetStaticParametersForMethod
                     let meth = provider.GetType().GetMethod( "GetStaticParametersForMethod", bindingFlags, null, [| typeof<MethodBase> |], null)  
                     if isNull meth then [| |] else
-                    let paramsAsObj = meth.Invoke(provider, bindingFlags ||| BindingFlags.InvokeMethod, null, [| box x |], null) 
+                    let paramsAsObj = 
+                        try meth.Invoke(provider, bindingFlags ||| BindingFlags.InvokeMethod, null, [| box x |], null) 
+                        with err -> raise (StripException (StripException err))
```

Comment:
Hmmm... I'm copying what we did elsewhere for now - perhaps its wrong but it seems ok for here?